src/ui.rs

```diff
@@ -558,7 +565,9 @@ fn on_help_about(window: &gtk::ApplicationWindow) {
     let about = AboutDialog::new();
     about.set_transient_for(Some(window));
     about.set_program_name("NeovimGtk");
-    about.set_version(Some(crate::GIT_BUILD_VERSION.unwrap_or(env!("CARGO_PKG_VERSION"))));
+    about.set_version(Some(
+        crate::GIT_BUILD_VERSION.unwrap_or(env!("CARGO_PKG_VERSION")),
+    ));
```

Comment:
Also, these changes don't seem related to the commit at hand

Reply:
That was a `rustfmt`, I believe, but I can revert it.

Reply:
Ah that's fine then, I'd probably put it into a separate commit at least (can just be something short like "Rustfmt"

Reply:
I've opted to just revert this because the code is not rustfmt clean at all and I've found that virtually every file will need to be touched.  We can do that in a separate PR.